src/graphics/mesh.rs

```diff
@@ -624,6 +644,16 @@ impl Mesh {
         self.buffer = vbuf;
         self.slice = slice;
     }
+
+    /// Returns a slice for this mesh that could be used for manual draw call submission
+    pub fn get_slice(&self) -> &gfx::Slice<gfx_device_gl::Resources> {
+        &self.slice
+    }
+
+    /// Returns a vertex buffer for this mesh that could be used for manual draw call submission
+    pub fn get_vertex_buffer(&self) -> gfx::handle::Buffer<gfx_device_gl::Resources, Vertex> {
```

Comment:
Are `get_slice()` and `get_vertex_buffer()` necessary?  I'm not sure I can think of a situation where I'd want to use them.

Reply:
They are, my distance field rendering is rendering to multiple render targets simultaneously. ggez currently does not support that natively, so I'm manually submitting draw calls to the gfx encoder, which requires access to the slice and vertex buffer.